src/js/configs.js

```diff
@@ -118,7 +118,7 @@ module.exports = {
     tls13: '1.13.0',
   },
   openssl: {
-    latestVersion: '1.1.1w',
+    latestVersion: '3.3.2',
```

Comment:
The 3.4.0 should now include additional DHEattack CVEs fixed, FYI:

```suggestion
    latestVersion: '3.4.0',
```